README.md

```diff
@@ -1,32 +1,122 @@
-GiniBank SDK Xamarin Bindings and Examples
-=================================================
-
-This repository contains projects to help you use the Android and iOS GiniBank SDK  with Xamarin.
-
-Requirements
------------
-
-##### Visual Studio Community 2019 for Mac
+# Gini Bank SDK Xamarin Bindings and Examples
+
+This repository contains projects to help you use the Android and iOS Gini Bank SDK with Xamarin.
+
+**Important:** If you are using the Gini Vision Library please consult the [Migrate from
+gini-vision-lib-xamarin](#migrate-from-gini-vision-lib-xamarin) section to update to the Gini Bank SDK.
+
+- [Gini Bank SDK Xamarin Bindings and Examples](#gini-bank-sdk-xamarin-bindings-and-examples)
+  - [New SDKs](#new-sdks)
+    - [Gini Bank SDK](#gini-bank-sdk)
+    - [Gini Capture SDK](#gini-capture-sdk)
+    - [Gini Bank API Library](#gini-bank-api-library)
+  - [Requirements](#requirements)
+    - [Visual Studio Community 2019 for Mac](#visual-studio-community-2019-for-mac)
+    - [iOS](#ios)
+    - [Android](#android)
+  - [Migrate from gini-vision-lib-xamarin](#migrate-from-gini-vision-lib-xamarin)
+    - [Android](#android-1)
+    - [iOS](#ios-1)
+  - [Android](#android-2)
+    - [Documentation](#documentation)
+    - [Binding Libraries](#binding-libraries)
+    - [How to use in new project](#how-to-use-in-new-project)
```

Comment:
"a" article is missing 😉

    - [How to use in a new project](#how-to-use-in-new-project)